compiler/tests/org/jetbrains/kotlin/code/CodeConformanceTest.kt

```diff
@@ -60,7 +60,10 @@ class CodeConformanceTest : TestCase() {
                 "libraries/tools/kotlin-test-nodejs-runner/node_modules",
                 "libraries/tools/kotlinp/src",
                 "libraries/tools/new-project-wizard/new-project-wizard-cli/build",
-                "out"
+                "out",
+                "kotlin-native/build",
+                "kotlin-native/performance",
+                "kotlin-native/samples"
```

Comment:
Looks like samples and performance should not be excluded, why did this test fail?

Reply:
Because of nature of the CodeConformanceTest and Co, if file is too big for test it start failing.